lib/services/configuration.go

```diff
@@ -71,10 +71,14 @@ type ClusterConfiguration interface {
 	// DeleteClusterNetworkingConfig deletes ClusterNetworkingConfig from the backend.
 	DeleteClusterNetworkingConfig(ctx context.Context) error
 
+	// GetInstallers gets all installer scripts from the backend
+	GetInstallers(context.Context) ([]types.Installer, error)
 	// GetInstaller gets the installer script from the backend
-	GetInstaller(context.Context) (types.Installer, error)
+	GetInstaller(context.Context, string) (types.Installer, error)
```

Comment:
Nit: Use named arguments when it's not clear what the argument is from its type.
```suggestion
	GetInstaller(ctx context.Context, name string) (types.Installer, error)
```